.github/workflows/test.yml

```diff
@@ -61,6 +66,18 @@ jobs:
       matrix:
         os: [ ubuntu-latest, macos-latest ]
         python-version: [ "3.8", "3.11" ]
+        pytest-split-group: [1, 2, 3]
```

Comment:
If we can use non integer identifiers for these groups that would be better, since I have no idea what group 4 means

Reply:
I can add comments. Pytest-split tries to divide the tests into n groups labeled 1, ..., n which have approximately the same total runtime. So they have to be integers.

Reply:
The runs are labeled like "test / ... (3, 5)" which is run number 3 out of 5.